runtime/src/main/java/org/corfudb/runtime/object/transactions/SnapshotTransactionalContext.java

```diff
@@ -2,10 +2,12 @@
 
 import com.google.common.collect.ImmutableSet;
 
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collections'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)